### PR TITLE
Readme dart ci bage is icorrect #153 #160

### LINKED ISSUE #153: Readme dart ci bage is icorrect
Readme dart ci bage is incorrect

It is currently 

`[dart CI](https://github.com/CyBear-Jinni/CBJ_Smart-Device/workflows/Dart%20CI/badge.svg)](https://github.com/CyBear-Jinni/CBJ_Smart-Device/actions?query=workflow%3A%22Dart+CI%22)`

But the repo name has changed from `CBJ_Smart-Device` to `cbj_smart-device`

### PR DESCRIPTION
**Pull request by BxtGeek**
changed from CBJ_Smart-Device to cbj_smart-device

### CODE REVIEW

**Comment by BxtGeek** [comment]
I added the PR to Dev.

File: README.md [hunk 1]
```diff
@@ -43,7 +43,7 @@ This part is using the snap store in order to have easy way to update all the de
 
 ### Smart device preparations:
 
-Setting up instructions for the new device can be found in the [CyBear Jinni Smart Device Resources](https://github.com/CyBear-Jinni/CBJ_Smart-Device_Resources) repository.
+Setting up instructions for the new device can be found in the [CyBear Jinni Smart Device Resources](https://github.com/CyBear-Jinni/cbj_smart-device_Resources) repository.
```
Comment (guyluz11) [inline README.md:46]: cbj_smart-device_Resources should stay CBJ_Smart-Device_Resources as this repo didn't changed his name

Comment (BxtGeek) [reply README.md:46]: Okay let me just change and confirm

Comment (BxtGeek) [reply README.md:46]: It is same. It does not changed

Comment (guyluz11) [reply README.md:46]: It should be `CBJ_Smart-Device_Resources`

**Comment by guyluz11** [comment]
Thanks @BxtGeek 

I will accept the pr as most of the URL is not case sensitive, it is not the same URL but it does work.

And a happy Hacktoberfest :D

**Pull request merged**